packages/core/README.md

```diff
@@ -81,6 +81,53 @@ The mapConfiguration allows controlling many client instance details.
 | stylePath | string? | If no link tag with `data-polar="true"` is found in the document, this path will be used to create the link node in the client itself. It defaults to `'./style.css'`. Please mind that `data-polar="true"` is deprecated since it potentially led to flashes of misstyled content. stylePath will replace that solution in the next major release. |
 | renderFaToLightDom | boolean? | POLAR requires FontAwesome in the Light/Root DOM due to an [unfixed bug in many browsers](https://bugs.chromium.org/p/chromium/issues/detail?id=336876). This value defaults to `true`. POLAR will, by default, just add the required CSS by itself. Should you have a version of Fontawesome already included, you can try to set this to `false` to check whether the versions are interoperable. |
 
+<details>
+<summary>Example configuration</summary>
```

Comment:
This looks really great in Markdown but is (weirdly enough) not properly translated to HTML when running any `docs` command (e.g. `npm run docs:afm`).

There is surely an adjustment needed in `makeDocs` to not get

```html
<p>&lt;details&gt;
&lt;summary&gt;Example configuration&lt;/summary&gt;</p>
```

as the resulting html.

Reply:
I went through the README files and fixed the indentation, and one of the examples that I now know had been false. 

For markdownIt to recognize the html tags I had to make some adjustments to the script creating the docs.

Please keep in mind that some of the examples might have mistakes in them that I don't know of (as I realized yesterday with the address search example for wfs). 


@dopenguin 🏓